src/components/policy/policy_external/include/policy/sql_pt_queries.h

```diff
@@ -117,27 +117,15 @@ extern const std::string kSelectApplicationFull;
 extern const std::string kUpdatePreloaded;
 extern const std::string kUpdateRemoteControlDenied;
 extern const std::string kSelectRemoteControlDenied;
-extern const std::string kDeleteAppGroupPrimaryByApplicationId;
```

Comment:
Why were all of these removed?

Reply:
@jacobkeeler it is related to device/passanger logic and should not be implemented in this proposal